gradle.properties

```diff
@@ -1,5 +1,5 @@
-ambariClientName=ambari-client20
-ambariClientVersion=2.0.27
+ambariClientName=ambari-client21
```

Comment:
I just tested the change with Ambari 2.0 and works the same so I moved the change to that branch as well. Will adjust a desired changes soon.